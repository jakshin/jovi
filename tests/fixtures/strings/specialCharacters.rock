say "foo \ bar"
say "foo \\ bar"
say "\t\u\r\n"
say "foo ${bar} \n"
say "foo \${bar} \n"
say "foo \\${bar} \n"
