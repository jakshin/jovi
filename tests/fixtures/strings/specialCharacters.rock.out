foo \ bar
foo \\ bar
\t\u\r\n
foo ${bar} \n
foo \${bar} \n
foo \\${bar} \n
